src/utils/i18nHelper.js

```diff
@@ -9,26 +6,9 @@ Vue.use(VueI18n);
 
 const fallbackLocale = 'en';
 
-export const i18n = new VueI18n({
+export default new VueI18n({
   locale: fallbackLocale,
   fallbackLocale,
   formatFallbackMessages: true,
   messages: { en },
 });
-
-export const langs = {
-  en: {
-    getMessages: () => require('../locales/en.json'),
-  },
-  zh: {
-    getMessages: () => require(/* webpackChunkName: "locale-zh" */ '../locales/zh.json'),
-  },
-};
-
-export const fetchAndSetLocale = async (languageCode) => {
-  if (!i18n.availableLocales.includes(languageCode)) {
-    const messages = (await langs[languageCode].getMessages());
-    i18n.setLocaleMessage(languageCode, messages);
-  }
-  i18n.locale = languageCode;
-};
```

Comment:
`require()` instead of `import()` doesn't have sense here. So, async import is broken, `fetchAndSetLocale` won't fetch anything, and as far it is not used I propose to remove it instead of fixing.

Reply:
It is used https://github.com/aeternity/superhero-ui/pull/532/ here. It is hanging PR for a long period of time and needs to be refreshed. Back then the only requirement missing were the translations.

Reply:
Let's add it back in that PR (while refreshing) and to polish it all together.